packages/ui/src/components/Molecules/ContactForm.stories.ts

```diff
@@ -0,0 +1,9 @@
+import { Meta, StoryObj } from '@storybook/react';
+
+import { ContactForm as Component } from './ContactForm';
+
+export default {
+  component: Component,
+} satisfies Meta<typeof Component>;
+
+export const Empty = {} satisfies StoryObj<typeof Component>;
```

Comment:
This would be a good opportunity to add [play functions](https://storybook.js.org/docs/writing-stories/play-function) to:

* Test for visual regressions in the filled form
* Test for visual regressions in error display
* Test field validation in general.

Also it is possible to mock executors of the mutation to have stories that show success and error messages. The [Content Hub stories](https://github.com/AmazeeLabs/silverback-template/blob/release/packages/ui/src/components/Organisms/ContentHub.stories.tsx) are a good example for this.

Reply:
the tests are there, the storybook play functions are working well, except the case when we need the mocked executors (the part that you want to refactor in SLB).